_config.yml

```diff
@@ -20,8 +22,8 @@ scripts: assets/js/es6-promise-auto-min.js
 
 # Author/Organization info to be displayed in the templates
 author:
-  name: Put name here
-  url: Put url here
+  name: British Columbia Mine Information
```

Comment:
Do we really want to fill these placeholders? I thought the jekyll-barebones project was a generic project that didn't belong to EAO or EMPR.

Reply:
Good point I'll clean this up